src/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -1030,6 +1030,51 @@ public void Connect(IPAddress[] addresses, int port)
             }
         }
 
+        public void Close()
+        {
+            if (s_loggingEnabled)
+            {
+                NetEventSource.Enter(NetEventSource.ComponentType.Socket, this, nameof(Close), null);
+            }
+            if (GlobalLog.IsEnabled)
+            {
+                GlobalLog.Print("Socket#" + LoggingHash.HashString(this) + "::Close() timeout = " + _closeTimeout);
+            }
+
+            Dispose();
+
+            if (s_loggingEnabled)
+            {
+                NetEventSource.Exit(NetEventSource.ComponentType.Socket, this, nameof(Close), null);
+            }
+        }
```

Comment:
I'm just copying the CLR here.  It makes a certain amount of sense; `Close(int)` has additional logging of the timeout value, so this keeps things nice and symmetrical.